src/components/Home.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react'
+import { defaultURL } from '../App'
+import ArticleCard from './ArticleCard'
+
+export default function Home() {
+  const [ articles, setArticles ] = useState([])
+  const [ loading, setLoading ] = useState(true)
+
+  useEffect(()=>{
+    setLoading(true)
+    defaultURL.get('/articles')
+    .then((response)=>{
+      return setArticles(response.data)
+    })
+    .then(()=>{
+      setLoading(false)
```

Comment:
This does not need a new .then block, it can go in the one above as the state of loading can be changed once articles are retrieved.